### PR TITLE
KetherNFTRenderer V2

### PR DESCRIPTION
**Pull request by shazow**
Wanted to try embedded svg background, could offload that static aspect to IPFS or whatever.

Preview:
image

### CODE REVIEW

File: static/nft-sample.svg [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    <text x="5" y="34">
+        Thousand Ether Homepage
+    </text>
+    <text x="1000" y="34" text-anchor="end">
+        #1234
+    </text>
+    <svg y="50" width="1000" height="1000" viewBox="0 0 100 100">
+        <rect width="100" height="100" fill="white" />
+        <image width="100" height="100" href="mainnet-grid.svg" opacity="0.1" />
```
Comment (mveytsman) [inline static/nft-sample.svg:17]: When testing locally this background image loads when I open the svg in my browser but doesn't when I embed it into an html page:

```html
<html>
  <body><img src="nft-sample.svg"></img></body>
</html>
```

This is using a local server not `file://` urls.

**Comment by mveytsman** [comment]
Screen Shot 2021-10-18 at 1 36 55 PM

I got a bit of a blur/shadow effect with a dropshadow filter on the outline:

```svg
 <filter id="dropShadow">
     <feDropShadow dx="0" dy="0" stdDeviation=".1"/>
 </filter>
```

and moving the outline a bit:

```svg
<svg y="50" width="1000" height="1000" viewBox="0 0 100 100">
      <rect width="100" height="100" fill="white" />
      <image width="100" height="100" href="mainnet-grid.svg" opacity="0.1" />
      <rect x="60" y="22" width="2" height="1" fill="rgb(66,185,131)"/>
      <rect x="59.5" y="21.5" width="3" height="2" fill="rgba(66,185,131,0.2)" filter="url(#dropShadow)"/>
  </svg>
```

**Comment by shazow** [comment]
image

Added some green around the white, shrug?

**Comment by shazow** [comment]
image

Same using stroke on the highlight element, rather than two elements (more visual overlap, less code)

File: test/kethernft.js [hunk 2]
```diff
@@ -133,7 +135,7 @@ describe('KetherNFT', function() {
     await KNFT.connect(account1).wrap(idx, await account1.getAddress());
     {
       const expected = {
-        "name": "ThousandEtherHomepage #0: 30x40 at [10,20]",
+        "name": "Ad #0: 30x40 at [10,20]",
```
Comment (shazow) [inline test/kethernft.js:138]: @mveytsman FYI changing the name in the metadata to be shorter so it doesn't get truncated in OpenSea UI.

Comment (shazow) [reply test/kethernft.js:138]: Oh looks like OpenSea made it wrap now, but still it's redundant with the same string just above it

image

File: components/App.vue [hunk 3]
```diff
@@ -65,17 +65,18 @@
...
-        <li v-if="$store.state.numNSFW > 0">
+        <li v-if="$store.getters.numNSFW > 0">
           <a v-if="!showNSFW" v-on:click="showNSFW = true"
-            >Show NSFW ({{ $store.state.numNSFW }})</a
+            >Show NSFW ({{ $store.getters.numNSFW }})</a
           >
           <a v-else v-on:click="showNSFW = false">Hide NSFW</a>
         </li>
         <li><a href="https://v1.thousandetherhomepage.com">Switch to v1 (2017)</a></li>
+        <li class="buttons">
```
Comment (shazow) [inline components/App.vue:75]: @mveytsman Ok merging this into prod?

File: store/index.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ export const state = () => {
     pixelsOwned: 0,
     grid: null, // lazy load
     previewAd: null,
-    gridVis: true,
+    vis: 'grid',
```
Comment (shazow) [inline store/index.js:16]: @mveytsman Give this some fresh :eyes: before I merge

**Pull request merged**